pallets/bridge/src/lib.rs

```diff
@@ -259,6 +259,22 @@ decl_error! {
     }
 }
 
+// A value placed in storage that represents the current version of the this storage. This value
+// is used by the `on_runtime_upgrade` logic to determine whether we run storage migration logic.
+// NB If you add a new value, don't forget to update the implementation of `Default`.
+#[derive(Encode, Decode, Clone, Copy, PartialOrd, Ord, PartialEq, Eq)]
+pub enum Version {
```

Comment:
It's ok for this PR but I'm not liking the code duplication. Let's think of something else later. Maybe use u8 + a max version number per module. Or a macro that defines the enum automatically based on the max number we pass to it.

Reply:
I would prefer that we use more meaningful versioning names, so numbers seem inappropriate (and generating a set of identifiers based on a number requires procedural macros. This isn't something you can do outside of will-probably-never-stable `concat_idents!` so no declarative macros.

The `Default` impl is something we could generate using https://crates.io/crates/derivative though, to slightly reduce duplication.

We could however define a macro like so:
```rust
macro_rules! version_enum {
    ($($v:ident)*) => {
        pub enum Version {
            Initial,
            $($v),*
        }

impl Default for Version {
    fn default() -> Self {
        Version::Initial
    }
}
    }
}
```

The initial version will always be present then, and you can nicely declare something like:
```rust
version_enum! { Foo, Bar, baz }
```
and you're done.